lib/results-list.js

```diff
@@ -131,35 +131,45 @@ export const ResultItem = styled.span.attrs(() => ({ 'data-module': 'ResultItem'
   width: 100%;
   font-size: var(--fontSizes-normal);
   color: var(--colors-primary);
-  background-color: ${props => props.isPrivate ? "var(--colors-private-background);" : "var(--colors-background);"};
+  background-color: ${(props) => (props.isPrivate ? 'var(--colors-private-background);' : 'var(--colors-background);')};
   position: relative;
-  padding: var(--space-1);
+  padding: 0;
   text-decoration: none;
-  margin: var(--space-1) 0;
+  margin: 0;
 
   &:focus,
   button:focus &,
   a:focus & {
     box-shadow: 0 0 0 1px white, 0 0 0 3px var(--colors-focused);
     outline-color: transparent;
     color: var(--colors-selected-text);
-    background-color: var(--colors-selected-background);
+    background-color: #5a78ff;
+    color: black;
 
     ${ResultDescription} {
-      color: var(--colors-selected-secondary);
```

Comment:
After removing this variable does this look right on light/dark mode?

Reply:
Ohhh good catch. The background color `#5a78ff` is the same in both in both light and dark mode so it should be fine, and I didn't see anything strange when I checked @keithk